system/views/admin_page.py

```diff
@@ -0,0 +1,30 @@
+import tkinter as tk
+from tkinter import ttk
+
+
+class AdminPage(tk.Frame):
```

Comment:
This should be called AdminView, same with LoginView, RegisterView etc. If you're wondering if its the reason why your imports aren't working, its not. I've started to get the same problem with completely different class files.